template_parts/Bids_modality_specific/Microscopy.json

```diff
@@ -0,0 +1 @@
+{"elabftw":{"extra_fields_groups":[{"id":1,"name":"Microscopy "}]},"extra_fields":{"BodyPart":{"type":"text","value":"","group_id":1,"position":34,"description":"Body part of the organ / region scanned (e.g. BRAIN).","blank_value_on_duplicate":false},"Immersion":{"type":"text","value":"","group_id":1,"position":29,"description":"Lens immersion medium.","blank_value_on_duplicate":false},"PixelSize":{"type":"text","value":"","group_id":1,"position":27,"description":"2- or 3-number array of the physical size of a pixel [X,Y,(Z)].","blank_value_on_duplicate":false},"Magnification":{"type":"number","value":"","group_id":1,"position":31,"description":"Lens magnification (e.g. 40).","blank_value_on_duplicate":false},"PixelSizeUnits":{"type":"select","value":"mm","options":["mm","um","nm"],"group_id":1,"description":"Unit format of the specified PixelSize."},"SampleFixation":{"type":"text","value":"","group_id":1,"position":39,"description":"Description of the tissue sample fixation.","blank_value_on_duplicate":false},"BodyPartDetails":{"type":"text","value":"","group_id":1,"position":35,"description":"Additional details about body part or location.","blank_value_on_duplicate":false},"SampleEmbedding":{"type":"text","value":"","group_id":1,"position":38,"description":"Description of the tissue sample embedding.","blank_value_on_duplicate":false},"NumericalAperture":{"type":"number","value":"","group_id":1,"position":30,"description":"Lens numerical aperture (e.g. 1.4).","blank_value_on_duplicate":false},"SampleEnvironment":{"type":"select","value":"","options":["in vivo","ex vivo","in vitro"],"group_id":1,"position":37,"description":"Environment in which the sample was imaged.","blank_value_on_duplicate":false},"BodyPartDetailsOntology":{"type":"text","value":"","group_id":1,"position":36,"description":"Ontology URI for BodyPartDetails.","blank_value_on_duplicate":false},"ImageAcquisitionProtocol":{"type":"text","value":"","group_id":1,"position":32,"description":"Description or URI of the image acquisition protocol.","blank_value_on_duplicate":false},"ChunkTransformationMatrix":{"type":"text","value":"","group_id":1,"position":29,"description":"3x3 or 4x4 affine transformation matrix describing spatial chunk transformation, for 2D and 3D respectively (e.g. [[2, 0, 0], [0, 3, 0], [0, 0, 1]] for 2D or [[1, 0, 0, 0], [0, 2, 0, 0], [0, 0, 3, 0], [0, 0, 0, 1]] for 3D). Non-spatial dimensions like time and channel are not included.","blank_value_on_duplicate":false},"OtherAcquisitionParameters":{"type":"text","value":"","group_id":1,"position":33,"description":"Other relevant image acquisition parameters.","blank_value_on_duplicate":false},"ChunkTransformationMatrixAxis":{"type":"text","value":"","group_id":1,"position":30,"description":"List of axis names corresponding to ChunkTransformationMatrix (e.g. [\"X\", \"Y\"] or [\"Z\", \"Y\", \"X\"]). REQUIRED if ChunkTransformationMatrix is present.","blank_value_on_duplicate":false}}}
```

Comment:
The format of this file does not appear like the other ones... Look at it: https://github.com/INT-NIT/elabforms_BIDSMetadata/blob/57539d0c4af256f0784e19bd1a112359e2d092ab/template_parts/Bids_modality_specific/Microscopy.json

Can you check?